### PR TITLE
Fix headers and hints

### PR DESCRIPTION
**Pull request by rgarner**
While making #45, we noticed a couple of problems:

1. A few headers had the wrong closing tags 
2. Some options to a helper were duplicated and causing a "last in wins" scenario. Fixing this necessitated an extra CSS change.

### CODE REVIEW

File: app/assets/stylesheets/barnardos/_textarea.scss [hunk 1]
```diff
@@ -62,6 +62,9 @@
 .textarea__hint {
   color: $secondary-text;
   display: block;
+  font-family: $font-family;
```
Comment (ztolley) [inline app/assets/stylesheets/barnardos/_textarea.scss]: will this change also be needed in textfield and other form controls?

**Comment by rgarner** [comment]
Made up-to-date, @ztolley

File: app/assets/stylesheets/barnardos/_textarea.scss (hunk 1 above)
Comment (rgarner) [inline app/assets/stylesheets/barnardos/_textarea.scss]: Not sure – I cargo-culted this from your other PR.

Comment (rgarner) [inline app/assets/stylesheets/barnardos/_textarea.scss]: @ztolley how would I tell? Everything *looks* ok?

Comment (ztolley) [inline app/assets/stylesheets/barnardos/_textarea.scss]: you'd have to make some fake markup in gallery or something then recreate setting the label to have a title like style

Comment (rgarner) [inline app/assets/stylesheets/barnardos/_textarea.scss]: Ok. I've made a *lot* of markup in gallery, and put some gash styling in the top so we can see what code applies to which examples. This covers things like hints.

Are there any CSS unit testing frameworks we should be considering? 😛

**Pull request merged**